besser/bot/library/event/event_library.py

```diff
@@ -25,3 +25,18 @@ def intent_matched(target_intent: Intent, matched_intent: Intent) -> bool:
         bool: True if the 2 intents are the same, false otherwise
     """
     return target_intent.name == matched_intent.name
+
+def session_operation_matched(current_value, operation: operator, target_value) -> bool:
```

Comment:
You could use a typehint in current_value and target_value: the Any type:

from typing import Any

(update also the docstring types with Any)

Then, you could check if current_value and target_value are the same type, to avoid throwing an exception here?